pkgs/os-specific/darwin/yabai/default.nix

```diff
@@ -1,17 +1,24 @@
-{ lib, stdenv, fetchFromGitHub, darwin, xxd }:
+{ lib, stdenv, fetchFromGitHub, runCommand, darwin, xxd }:
 
+let
+  buildSymlinks = runCommand "yabai-build-symlinks" { } ''
+    mkdir -p $out/bin
+    ln -s /usr/bin/xcrun /usr/bin/xcodebuild /usr/bin/tiffutil /usr/bin/qlmanage $out/bin
```

Comment:
That won't work with the sanbox.

Reply:
I didn't know about the sandbox concept.

The [`macvim`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/vim/macvim.nix) package also uses a very similar configuration to yabai. Would it be ok if I apply the `sandboxProfile` like the one below?

```nix
{
  sandboxProfile = ''
    (allow file-read* file-write* process-exec mach-lookup)
    ; block homebrew dependencies
    (deny file-read* file-write* process-exec mach-lookup (subpath "/usr/local") (with no-log))
  '';
}
```

Reply:
Hmmmm

We can't build it with the tools in nixpkgs, can we?

Reply:
When I tried, it couldn't build with other nixpkgs tools like `xcbuild`.

Reply:
@NixOS/darwin-maintainers

Reply:
> We can't build it with the tools in nixpkgs, can we?

I tried to make it work, but there are a few problems that stopped me.

* SDK: x86_64-darwin SDK is too old. The minimum supported version of macOS is 10.13.
* clang: requires `ptrauth.h` (arm64e), which is still in the process of being upstreamed (see [D112941][1]) as far as I can tell.

There’s another issue. Yabai requires entitlements. There’s not no good way to do that right now in nixpkgs. The way to go is probably to use upstream’s binary like was down for iTerm.

[1]: https://reviews.llvm.org/D112941